index.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
+const fp = require('fastify-plugin')
+const symbols = require('fastify/lib/symbols')
+const Express = require('express')
+const kMiddlewares = Symbol('fastify-express-middlewares')
+const kExpress = Symbol('fastify-express-instance')
+
+function expressPlugin (fastify, options, next) {
+  // TODO: we should use decorate, but `use` is already
+  // a public API of Fastify. In Fastify v3 it will be deprecated,
+  // so we will able to use `decorate`
+  fastify.use = use
```

Comment:
Once https://github.com/fastify/fastify/pull/2014 gets merged, I'll update this one as well.